network/src/peer_store/sqlite/db.rs

```diff
@@ -21,62 +21,57 @@ pub fn create_tables(conn: &Connection) -> DBResult<()> {
     score INTEGER NOT NULL,
     status INTEGER NOT NULL,
     endpoint INTEGER NOT NULL,
-    ban_time INTEGER NOT NULL,
-    last_connected_at INTEGER NOT NULL
+    ban_time_ms INTEGER NOT NULL,
+    last_connected_at_ms INTEGER NOT NULL
     );
     "#;
     conn.execute_batch(sql)?;
     let sql = r#"
     CREATE TABLE IF NOT EXISTS peer_addr (
     id INTEGER PRIMARY KEY NOT NULL,
-    peer_info_id INTEGER NOT NULL,
+    peer_id BINARY NOT NULL,
```

Comment:
Will the schema be migrated automatically or we have to delete the old database file manually?

Reply:
We have deleted it manually